doc/api/fs.markdown

```diff
@@ -576,6 +576,9 @@ These stat objects are instances of `fs.Stat`.
 If you want to be notified when the file was modified, not just accessed
 you need to compare `curr.mtime` and `prev.mtime`.
 
+_Note: when an `fs.watchFile` operation results in an `ENOENT` error, it will
+invoke the callback once. This is a change in functionality since v0.10._
+
```

Comment:
Since it is an instance of [`fs.Stat`](https://iojs.org/api/fs.html#fs_class_fs_stats), I think that would belong there, rather than in the `fs.watchFile` docs. Perhaps this should be done in a separate PR?